### ISSUE TITLE
Tick DSPtach

### ISSUE THREAD

**Issue by eliodilar** [cross-platform/dspatch#16]
Hello,

i haven't find forum so i post my question here.
just to be sure

1) We can choose for component all component declaration, 2 tick modes:
Series: unthreading, sequential.
Parallel: Threading, parallelise

Isn't it?

Cordially

**Comment by MarcusTomlinson** [comment]
Right, what this is referring to is the tick mode. That is, how do you want the tick to be performed across the circuit.
- Series mode will tell the circuit to process components one-by-one in a single thread.
  - This mode will improve performance if your circuit has no parallel branches.
- Parallel mode will spawn a thread per component in the circuit.
  - This mode will improve performance if your circuit has parallel branches.

Note that there is another threading model at play in DSPatch too: buffering. Setting the buffer count on your circuit to something other than 1 can improve its performance when processing streams of data. Buffering can be used regardless of the tick mode selected.
